app/controllers/procurement_areas_controller.rb

```diff
@@ -1,6 +1,10 @@
 class ProcurementAreasController < ApplicationController
   def index
-    @procurement_areas = ProcurementArea.ordered_by_name
+    @procurement_areas = procurement_areas_ordered_by_name
```

Comment:
There was a reason, since I don't remember it, it was probably due to some interim step before arriving at this organisation of code. I'll remove it an push.